__test__/unit/types/interest_rate.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { InterestRate } from "../../../src/types/interest_rate";
+import { BigNumber } from "../../../utils/bignumber";
+
+describe("InterestRate", () => {
+    describe("instantiation", () => {
+        const percent = 12.4;
+        const rawAmount = new BigNumber(percent);
+
+        describe("#fromPercent", () => {
+            const interestRate = InterestRate.fromPercent(percent);
+
+            test("should expose the interest rate in raw form", () => {
```

Comment:
nit: can we use `it` instead of `test`?  A small difference, but makes using `it.only` while debugging much easier :)

Reply:
they're aliases. what's your reasoning. test feels native to jest.